http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -874,18 +838,15 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       autoscalingConfig = None
     )
 
-    // regionName can be empty in some test configurations
-    val regionName = if (azureRegionOpt.isEmpty) "" else azureRegionOpt.map(_.name).getOrElse("")
     for {
       nodepool <- defaultNodepool
       defaultClusterName <- KubernetesNameUtils.getUniqueName(KubernetesClusterName.apply)
-      clusterName = aksClusterName.map(c => KubernetesClusterName(c.value)).getOrElse(defaultClusterName)
     } yield SaveKubernetesCluster(
       cloudContext = cloudContext,
-      clusterName = clusterName,
+      clusterName = defaultClusterName,
```

Comment:
AKS's cluster name is always the same becuz there will only be one cluster per LZ?

Reply:
Yeah there's 1 cluster per LZ. And actually `defaultClusterName` will be a random string but not the actual cluster name… this is another place where we could update the Leo DB in back Leo after we query the LZ. (But again, Leo does not rely on this field for Azure.)